src/main/java/com/philderbeast/prisonpicks/Pick.java

```diff
@@ -200,7 +213,7 @@ static ItemStack getDrop(int enchantmentLevel, Block block, ItemStack tool)
 
         return drop;
     }
-
+    String tpick = "";
```

Comment:
more random class variables?

what is this? why dosent it have an accessor label, why is it down here?